arch/src/main/java/knaufdan/android/arch/databinding/livedata/LiveDataMerging.kt

```diff
@@ -7,23 +7,23 @@ import androidx.lifecycle.MediatorLiveData
  * Uses a [merging] function to combine the value of two [LiveData] sources into a result value which is observed by a created [LiveData] target.
  * Each time one source value changes, the [merging] functions is called to determine the new value which is posted onto the target.
  *
- * @param TargetData the type of data hold by the returned [LiveData]
+ * @param TargetType the type of data hold by the returned [LiveData]
```

Comment:
`Target` , `FirstSource` and `SecondSource` should be sufficient as it is a Generic and therefore describing a type by its nature

Reply:
done.